geotrellis-sentinelhub/src/test/scala/org/openeo/geotrellissentinelhub/MadeToMeasureCatalogApiTest.scala

```diff
@@ -24,9 +24,9 @@ class MadeToMeasureCatalogApiTest {
     val actualFeatures = catalogApi.search(collectionId, geometry, geometryCrs, from, to, accessToken, queryProperties)
 
     val expectedFeatures = Map(
-      "0" -> Feature(geometry, from),
-      "1" -> Feature(geometry, from plusDays 1),
-      "2" -> Feature(geometry, from plusDays 2)
+      "0" -> Feature(geometry, FeatureData(from, None)),
+      "1" -> Feature(geometry, FeatureData(from plusDays 1, None)),
+      "2" -> Feature(geometry, FeatureData(from plusDays 2, None))
```

Comment:
You might want to give a name things like None, null, true, false etc to improve readability e.g.


```scala
val selfUrl = None
...
"0" -> Feature(geometry, FeatureData(from, selfUrl))
```

Reply:
Fixed in https://github.com/Open-EO/openeo-geotrellis-extensions/pull/134